### PR TITLE
Fix alacritty un/install

### PR DESCRIPTION
**Pull request by darwingr**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [ ] The commit message includes the cask’s name and version.
       **NO VERSION NUMBER. This isn't a version update.**
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [ ] Named the cask according to the [token reference].
- [ ] `brew cask install {{cask_file}}` worked successfully.
- [ ] `brew cask uninstall {{cask_file}}` worked successfully.
- [ ] Checked there are no [open pull requests] for the same cask.
- [ ] Checked the cask was not [already refused].
- [ ] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

### CODE REVIEW

File: Casks/alacritty.rb [hunk 1]
```diff
@@ -8,9 +8,9 @@
   homepage 'https://github.com/alacritty/alacritty/'
 
   app 'Alacritty.app'
+  binary "#{appdir}/Alacritty.app/Contents/MacOS/alacritty"
 
   zap delete: [
                 '~/Library/Saved Application State/io.alacritty.savedState',
-                '~/.config/alacritty',
```
Comment (vitorgalvao) [inline Casks/alacritty.rb:14]: Why are you removing this?

File: Casks/alacritty.rb [hunk 2]
```diff
@@ -8,9 +8,9 @@
   homepage 'https://github.com/alacritty/alacritty/'
 
   app 'Alacritty.app'
+  binary "#{appdir}/Alacritty.app/Contents/MacOS/alacritty"
```
Comment (vitorgalvao) [inline Casks/alacritty.rb:11]: Unless there’s official documentation recommending this, we don’t do it.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/alacritty.rb (hunk 2 above)
Comment (darwingr) [reply Casks/alacritty.rb:11]: The is okay to execute directly on MacOS and other platforms.
For example, here's the output you actually get with `-h`:
image


Here's the documentation on how to use alacritty on the command line to make it easier to configure your keyboard mappings: https://github.com/alacritty/alacritty/wiki/Keyboard-mappings

File: Casks/alacritty.rb (hunk 1 above)
Comment (darwingr) [reply Casks/alacritty.rb:14]: Sorry, I put it in the commit message.
This file is never created by the app. The only way it is ever created is intentionally/deliberately by the user. If that's the case they likely would not want it deleted along with the other trash.

The same approach would go for MacVim, Wireshark, Emacs, GnuCash, etc,  to delete their respective user configs found in `~/.config/` and `~/`.

**Pull request merged**